### PR TITLE
KenCarp47 & KenCarp58 Tests

### PR DESCRIPTION
**Pull request by utkarsh530**
Hi,
I have added a bunch of tests for new KenCarp methods. Please have a look.
@ChrisRackauckas @kanav99

### CODE REVIEW

File: test/regression/iipvsoop_tests.jl [hunk 1]
```diff
@@ -16,6 +16,10 @@ sol5 = solve(prob,KenCarp4(),abstol=1e-12,reltol=1e-12)
 sol6 = solve(prob,KenCarp4(autodiff=false),abstol=1e-12,reltol=1e-12)
 sol7 = solve(prob,KenCarp4(autodiff=false,diff_type=Val{:central}),abstol=1e-12,reltol=1e-12)
 sol8 = solve(prob,KenCarp4(autodiff=false,diff_type=Val{:complex}),abstol=1e-12,reltol=1e-12)
+sol5 = solve(prob,KenCarp47(),abstol=1e-12,reltol=1e-12)
+sol6 = solve(prob,KenCarp47(autodiff=false),abstol=1e-12,reltol=1e-12)
+sol7 = solve(prob,KenCarp47(autodiff=false,diff_type=Val{:central}),abstol=1e-12,reltol=1e-12)
+sol8 = solve(prob,KenCarp47(autodiff=false,diff_type=Val{:complex}),abstol=1e-12,reltol=1e-12)
```
Comment (kanav99) [inline test/regression/iipvsoop_tests.jl:23]: We can have similar autodiff tests for KenCarp58

**Comment by YingboMa** [comment]
Test failures look unrelated.

**Comment by ChrisRackauckas** [comment]
@YingboMa could you look into that? It's probably due to Roger's recent change.

**Comment by utkarsh530** [comment]
@ChrisRackauckas I re-ran Travis. Tests failure has been resolved; failure due to stalled build.

**Comment by ChrisRackauckas** [comment]
Let's merge, but can you figure out what the build stall is from an add a print statement so that it doesn't occur?

**Comment by utkarsh530** [comment]
Yes, I will try to find it.

**Comment by utkarsh530** [comment]
@ChrisRackauckas after looking in the successful build of `Algconvergence_III` here https://travis-ci.org/github/SciML/OrdinaryDiffEq.jl/jobs/699570947, it seems that `split_methods_tests` take more time than before due addition of tests from new KenCarps. The time previously before any KenCarp into the split tests was 517 s ≈ 8.6 mins and after adding KenCarp58 become 587s ≈ 9.8 mins. Thus after adding both of them it must have crossed the threshold of 10 mins.

**Comment by ChrisRackauckas** [comment]
so just add some print statements.

**Pull request merged**